src/Models/Author.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Portable\FilaCms\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Author extends Model
+{
+    protected $fillable = [
+        'first_name',
+        'last_name',
+        'is_individual',
+    ];
+
+    protected $casts = [
+        'is_individual' => 'boolean',
+    ];
+
+    public function getDisplayNameAttribute()
```

Comment:
Can you please use the Laravel attribute object (new way of doing attributes)? https://laravel.com/docs/10.x/eloquent-mutators#accessors-and-mutators

Reply:
Got it